src/users.ts

```diff
@@ -0,0 +1,874 @@
+import Errors from './errors'
+
+export interface User {
+  name?: string
+  id?: string
+  credentials?: {
+    identity: string
+    secret?: string
+  }
+  owner?: string
+  tags?: [string, string]
+  role?: string
+  status?: 'enabled' | 'disabled'
+  createdAt?: string
+  updatedAt?: string
+  updatedBy?: string
+}
+
+interface PageRes {
+  total: number
+  offset: number
+  limit: number
+}
+
+interface UsersInterface {
+  users: User[]
+  page: PageRes
+}
+
+interface Groups {
+  groups: Groups[]
```

Comment:
This is not correct since this should be an array of values of type `Group` not `Groups`. Meaning the `Group` type should be imported from the `groups.ts` file. Same case for things and channels. 

Also check the list user things/channels/groups functions, they don't seem to be functioning. I am getting this error:
```
SyntaxError: Unexpected non-whitespace character after JSON at position 4
    at JSON.parse (<anonymous>)
    at parseJSONFromBytes (node:internal/deps/undici/undici:5329:19)
    at successSteps (node:internal/deps/undici/undici:5300:27)
    at fullyReadBody (node:internal/deps/undici/undici:1447:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async specConsumeBody (node:internal/deps/undici/undici:5309:7)
```